test/evaluate/test_evaluate.py

```diff
@@ -35,6 +35,8 @@ def setup_class(cls):
             'other_files': []
         })
 
+        # TODO figure out why the input-nodes file is not being included in the data.pickle file
```

Comment:
There is an error where the input-nodes files is not being added

Reply:
#379 fixes the core of the issue: I don't have a good way to emit a warning without having to manually iterate through the entire dataframe, unfortunately.

Reply:
> Warn the user when they specify too many columns per any row (is this an option in pandas?)
We can check the length of the columns to make sure it's not past a certain number for a simple solution.

Reply:
It was a very strange error - it was because there were too many tabs in the first (non-header) row for the node table, which made pandas eagerly parse the `NODEID` column as the index column.